app/templates/index.html

```diff
@@ -3,7 +3,7 @@
 
 <div class="mx-5 my-2">
 	<div class="row my-3">
-		<form class="form d-flex" method="POST">
+		<form class="form d-flex" action="{{ url_for('search') }}" method="POST">
```

Comment:
this is fine, but making a note that this will be another "bit" that will need to be changed when i get aruond to the PR of unit tests

Reply:
Since merging, you will now need to change url_for('search') to url_for('main.search')